api/liquidate.ts

```diff
@@ -55,6 +55,8 @@ const liquidationHandler = async function () {
                 process.env.FEE_RECEIVER_ADDRESS
             );
             const receipt = await tx.wait();
+
+            liquidations.inc({ number: positions.length });
```

Comment:
This will count how many times each positions.length is executed.

eg there will be
trs_liquidations_success(SOME_COUNTING_NUMBER, POSITIONS.LENGTH_A)
trs_liquidations_success(SOME_COUNTING_NUMBER, POSITIONS.LENGTH_B)

for each POSITIONS.LENGTH it will count

Reply:
Ah I see.  I've put it in a for loop so it should just give the total number of liquidations

Reply:
Sorry, just had a close look. So
If it is just incrementing, you can increment multiple like https://github.com/siimon/prom-client#counter.
But, if you add labels it segregates them based on there label
so 
```
liquidations.inc({ name: "Label one"}, amount);
liquidations.inc({ name: "Label two"}, amount);
```
will track two counters and increment them seperately. Added to this PR and also added a Gauge metric to track last synced block